src/NUnitEngine/Addins/addin-tests/VisualStudioProjectLoaderTests.cs

```diff
@@ -56,6 +56,45 @@ public void ValidExtensions(string project, bool isGood)
         }
 
         [Test]
+        public void CheckExtensionAttribute()
+        {
+            Assert.That(typeof(VisualStudioProjectLoader),
+                Has.Attribute<ExtensionAttribute>());
+        }
+
+        // Note for review:
+        // The following test doesn't pass because AttributeConstraint always uses the
+        // first attribute it finds. Should we fix the syntax or just document it?
```

Comment:
This sounds like one we should fix, but not now. Does `AttributeConstraint` just work on the first attribute, or the first attribute of an attribute type? Either way, I think we should fix it. Can you enter an issue?